apps/studio/src/schemas/site.ts

```diff
@@ -3,3 +3,8 @@ import { z } from "zod"
 export const getConfigSchema = z.object({
   id: z.number().min(1),
 })
+
+export const setNotificationSchema = z.object({
+  id: z.number().min(1),
+  notificationStr: z.string(),
```

Comment:
I think just calling it notification is enough, wdyt? 

Also, id is abit ambiguous here but since narrow usage context, still ok with it. I do have to ask - how do we identify what site/resource/user a particular notification belongs to?

Lastly, this seems to suggest we allow empty notifications! Could I double check this assumption?

Reply:
1. Sure, renaming it
2. I followed the convention in the other methods, we seem to use id to indicate siteId. But agreed, I think it should be explicit, will rename. We can rename the other schemas in a separate PR
3. I thought to allow it, but on hindsight, we should enforce a min validation of at least 1 char for practical usage